hw03_frequency_analysis/top.go

```diff
@@ -1,6 +1,42 @@
 package hw03_frequency_analysis //nolint:golint,stylecheck
 
-func Top10(_ string) []string {
-	// Place your code here
-	return nil
+import (
+	"regexp"
+	"sort"
+	"strings"
+)
+
+func Top10(in string) []string {
+	in = regexp.MustCompile(`\n`).ReplaceAllString(in, " ")
+	in = regexp.MustCompile(`\t`).ReplaceAllString(in, "")
+	in = regexp.MustCompile(`\s\s+`).ReplaceAllString(in, " ")
+
+	if in == "" {
+		return make([]string, 0)
```

Comment:
return nil

https://github.com/uber-go/guide/blob/master/style.md#nil-is-a-valid-slice